DelvUI/Config/Attributes/ConfigTypeAttributes.cs

```diff
@@ -149,6 +149,19 @@ public CollapseWithAttribute(int pos, int id)
         }
     }
 
+    [AttributeUsage(AttributeTargets.Field)]
+    public class DragDropHorizontalAttribute : Attribute
+    {   
+        public string friendlyName;
+        public string[] names;
+
+        public DragDropHorizontalAttribute(string friendlyName, string[] names)
```

Comment:
It'd be great if you can add the _params_ keyword to the _names_ parameter, which allows to use the attribute like thiis:
`[DragDropHorizontal("Friendly Name", "Name1", "Name2"....)]`

You can check the **ComboAttribute** for an example.

Reply:
Thanks for the suggestion. I've implemented this change now.